### ISSUE TITLE
Make default column sort order in html configurable

### ISSUE THREAD

**Issue by qu1ck** [openscopeproject/InteractiveHtmlBom#124]

**Comment by Halbelunge** [comment]
Is there any possibilty to sort the columns at the moment?

**Comment by qu1ck** [comment]
Yes, you can sort columns by clicking on the header. But it will reset on page reload and you can not specify it at bom generation time. This bug is about fixing the latter.

**Comment by Halbelunge** [comment]
Maybe "sort" was the wrong word. I do not want to sort the rows content regarding to one column, I want to sort the order of the columns from left to right. So as an example change "value" with "quantity".

I found a way to modify the html afterwards, but would be nice to do this in the GUI. I think to have the extra fields at the end, not in the middle would be better :) (personal oppinion)

**Comment by qu1ck** [comment]
Not implemented yet, #123 covers that.

**Comment by Halbelunge** [comment]
Thank you :)

For the moment I know how to reorder manually :)

kepp going with this great project!
